strings.qmd

```diff
@@ -635,21 +632,21 @@ If you don't already know the code for your language, [Wikipedia](https://en.wik
 
 Base R string functions automatically use the locale set by your operating system.
 This means that base R string functions do what you expect for your language, but your code might work differently if you share it with someone who lives in a different country.
-To avoid this problem, stringr defaults to using English rules by using the "en" locale and requires you to specify the `locale` argument to override it.
+To avoid this problem, stringr defaults to English rules by using the "en" locale and requires you to specify the `locale` argument to override it.
 Fortunately, there are two sets of functions where the locale really matters: changing case and sorting.
 
-The rules for changing cases are not the same in every language.
-For example, Turkish has two i's: with and without a dot, and it capitalizes them in a different way to English:
+The rules for changing cases differ among languages.
+For example, an i without a dot (i.e., ı) is capitalized differently in English or Turkish:
```

Comment:
This is not quite correct since Turkish truly has two i's (two different letters in the alphabet).